Succulent/Classes/Succulent.swift

```diff
@@ -77,6 +85,18 @@ public class Succulent : NSObject, URLSessionTaskDelegate {
         }
     }
     
+    private init(configuration: Configuration?) {
+        super.init()
+        if let configuration = configuration, let ignoreVersioningRequests = configuration.ignoreVersioningRequests {
```

Comment:
We only run the `if` block if _both_ `configuration` and `configuration.ignoreVersioningRequests` are non-nil. This causes a fault when `configuration.ignoreVersioningRequests` is `nil` but `configuration.ignoreParameters` or `configuration.port` are not.